content/sensu-go/6.2/operations/monitoring-as-code/_index.md

```diff
@@ -0,0 +1,151 @@
+
+You can use any source control repository for maintaining your monitoring workflow configuration files.
+Your monitoring as code repository should include configuration files with the resource definitions you use in your monitoring workflow.
+
+Use `sensuctl dump` to [export some or all of your existing Sensu resource definitions][7] to standard out (STDOUT) or to a file.
+
+You can also use sensuctl commands to retrieve your Sensu resource definitions as you create them so you can copy them into your configuration files.
+For example, to retrieve the definition for a check named `check-cpu`:
+
+{{< language-toggle >}}
+
+{{< code shell "YML" >}}
+sensuctl check info check-cpu --format yaml
+type: CheckConfig
+api_version: core/v2
+metadata:
+  created_by: admin
+  name: check-cpu
+  namespace: default
+spec:
+  check_hooks: null
+  command: check-cpu.rb -w 75 -c 90
+  env_vars: null
+  handlers:
+  - slack
+  high_flap_threshold: 0
+  interval: 60
+  low_flap_threshold: 0
+  output_metric_format: ""
+  output_metric_handlers: null
+  proxy_entity_name: ""
+  publish: true
+  round_robin: false
+  runtime_assets:
+  - cpu-checks-plugins
+  - sensu-ruby-runtime
+  secrets: null
+  stdin: false
+  subdue: null
+  subscriptions:
+  - system
+  timeout: 0
+  ttl: 0
+{{< /code >}}
+
+{{< code shell "JSON" >}}
+sensuctl check info check-cpu --format json
+{
+  "command": "check-cpu.rb -w 75 -c 90",
+  "handlers": [
+    "slack"
+  ],
+  "high_flap_threshold": 0,
+  "interval": 60,
+  "low_flap_threshold": 0,
+  "publish": true,
+  "runtime_assets": [
+    "cpu-checks-plugins",
+    "sensu-ruby-runtime"
+  ],
+  "subscriptions": [
+    "system"
+  ],
+  "proxy_entity_name": "",
+  "check_hooks": null,
+  "stdin": false,
+  "subdue": null,
+  "ttl": 0,
+  "timeout": 0,
+  "round_robin": false,
+  "output_metric_format": "",
+  "output_metric_handlers": null,
+  "env_vars": null,
+  "metadata": {
+    "name": "check-cpu",
+    "namespace": "default",
+    "created_by": "admin"
+  },
+  "secrets": null
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+Many of our [guides][6] demonstrate how to create, update, and retrieve resource definitions in this way.
+
+## Implement a monitoring as code workflow
+
+There's no one "correct" way to implement monitoring as code.
+To get started, try [SensuFlow][5], our git-based approach to managing Sensu resources.
+
+## Best practices for monitoring as code
+
+The repository of resource definitions can be any manner of source control repository (git, subversion, etc.). While a specific directory structure is not required, we will be suggesting one later in this document.
+
+- To maintain consistency, save all of your resources as only one file type: YAML or JSON.
+- Include all dependencies within a resource definition.
+For example, if a handler requires a dynamic runtime asset and a secret, include the asset and secret definitions with the definition for the handler itself.
```

Comment:
These are cabbaged from Jef and Todd's blog post about SensuFlow. Are there other best practices to mention?

Maybe this section should include the content I currently have above under "Implement a monitoring as code workflow"